base/strings/string_util.h

```diff
@@ -47,7 +50,7 @@ inline int snprintf(char* buffer,
                     ...) {
   va_list arguments;
   va_start(arguments, format);
-  int result = vsnprintf(buffer, size, format, arguments);
+  int result = ::vsnprintf(buffer, size, format, arguments);
```

Comment:
This shouldn't be prefixed with `::` as we want it to call base::vsnprintf

Reply:
Fixed.